lib/everest.cpp

```diff
@@ -181,8 +181,21 @@ json Everest::call_cmd(const std::string& requirement_id, const std::string& cmd
     if (this->validate_data_with_schema) {
         json_validator validator(Config::loader, Config::format_checker);
         for (auto const& arg_name : arg_names) {
-            validator.set_root_schema(cmd_definition["arguments"][arg_name]);
-            validator.validate(json_args[arg_name]);
+            try {
+                validator.set_root_schema(cmd_definition["arguments"][arg_name]);
+                validator.validate(json_args[arg_name]);
+            } catch (const std::exception& e) {
+                std::ostringstream oss;
+                oss << "Call to "
+                    << this->config.printable_identifier(connection["module_id"], connection["implementation_id"])
+                    << "->" << cmd_name << "(";
+                for (auto const& key : arg_names) {
+                    oss << key << ",";
```

Comment:
this will leave an additional ',' at the end, probably we should refactor this exception handling code into separate functions